packages/@react-native-windows/tester/src/js/examples-win/Flyout/FlyoutExample.windows.tsx

```diff
@@ -251,7 +251,7 @@ class FlyoutExample extends React.Component<{}, IFlyoutExampleState> {
 }
 
 export const displayName = (_undefined?: string) => {};
-export const title = '<Flyout>';
+export const title = 'Flyout';
```

Comment:
add category?

Reply:
Touch on it briefly in the description. Without a category, the default is to say "Other". Can change them to "UI" if we want proper categories.

Reply:
Added categories to all of our custom examples